pkg/resource/nodegroup/sdk.go

```diff
@@ -297,6 +297,10 @@ func (rm *resourceManager) sdkFind(
 	}
 
 	rm.setStatusDefaults(ko)
+	if r.ko.Spec.ScalingConfig != nil && r.ko.Spec.ScalingConfig.DesiredSize != nil {
+		ko.Status.DesiredSize = r.ko.Spec.ScalingConfig.DesiredSize
+	}
```

Comment:
do we want the latest one in status instead?
```suggestion
	if ko.Spec.ScalingConfig != nil && ko.Spec.ScalingConfig.DesiredSize != nil {
		ko.Status.DesiredSize = ko.Spec.ScalingConfig.DesiredSize
	}
```

Reply:
good catch!